Gruntfile.js

```diff
@@ -0,0 +1,22 @@
+module.exports = function (grunt) {
+
+    "use strict";
+
+    grunt.initConfig({
+        buster: {
+            all : {
+                test: {
+                    config: "test/buster.js"
+                },
+                server: {
+                    port: 1112
+                }
+            }
+        }
+    });
+
+    grunt.loadNpmTasks("grunt-buster");
+
+    grunt.registerTask("test", ["buster"]);
+    grunt.registerTask("default", "test");
+};
```

Comment:
I'll remove it again, I don't need it any longer (since I am now starting buster-server myself from ci-test.sh)